doc/deployment.md

```diff
@@ -0,0 +1,126 @@
+Deployment of Hummingbird with Ansible
+======================================
+
+Prerequisites
+-------------
+
+The documentation below assumes:
+
+  1.  Ansible is installed on an admin machine or vm that has access to all nodes in the cluster.
+  2.  The user that you will be runing the ansible script as has SUDO access to all nodes in the cluster.
+
+Example Configuration
+---------------------
+
+The following configuration is used for the examples in this documentation:
+
+| Server | IP | SATA Devices | SSD Devices |
+| ------ | -- | ------------ | ----------- |
+| storage1 | 10.1.1.10 | /dev/sdd, /dev/sde, /dev/sdf, /dev/sdg | /dev/sdb, /dev/sdc | 
+| storage2 | 10.1.1.11 | /dev/sdd, /dev/sde, /dev/sdf, /dev/sdg | /dev/sdb, /dev/sdc | 
+| storage3 | 10.1.1.12 | /dev/sdd, /dev/sde, /dev/sdf, /dev/sdg | /dev/sdb, /dev/sdc |
+
+Configuring the Ring
+--------------------
+
+The ring is managed outside of Ansible with the `hummingbird` command line.  Ansible will distribute the ring files to the nodes of the cluster.  The ring files will need to be stored in `/etc/hummingbird` on the admin node:
+
+  1.  Create the etc dir for the ring: `sudo mkdir -p /etc/hummingbird`
+  2.  Set the permissions: `sudo chown -R $USER.$USER /etc/hummingbird`
+
+It can be useful to create a script to create the initial account, container and object rings.  When configuring the ring, a good starting point is to use `3` replicas, a part power of `22`, and a min part hours of `162` for most clusters.  The following example also uses the SATA devices for the object servers and the SSD devices for the account and container servers.  For more advanced info on using the ring, see the Hummingbird documentation.
+
+`make_rings.sh`:
+```
+#!/bin/bash
+
+cd /etc/hummingbird
+
```

Comment:
so i guess add some notes that ansible will partition / format the raw devices and will name them the same as they are named on disk with a 1 on the end? so you can use that in the ring.